packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -254,6 +254,18 @@ new lambda.NodejsFunction(this, 'my-handler', {
 
 Note: A [`tsconfig.json` file](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html) is required 
 
+## Choosing which version of a dependency to bundle
```

Comment:
Let's simply add the option in the section [Configuring `esbuild`](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-lambda-nodejs/README.md#configuring-esbuild)

something like `mainFields: ['module', 'main'], // <short description and defaults to>`